File: techgig/tom_and_jerry/TomAndJerry.py
def find(tj, res):
    i = tj[2]
    while i > 0:
        t = i - tj[0]
        if t > 0:
            i = t + tj[1]
        else:
            break
        res += 1
    return res


n = int(input())
for i in range(n):
    tj = list(map(int, input().split()))
    print(find(tj,1))

File: techgig/tom_and_jerry/TomAndJerry.cpp
#include <stdio.h>
#include <iostream>
using namespace std;

 int find(int tj[], int res) {
    for (int i = tj[2]; i > 0; ) {
      int t = i - tj[0];
      if (t > 0) {
        i = t + tj[1];
      } else break;
      res++;
    }
    return res;
  }
  
int main() {
    int n;
    scanf("%d", &n);
    int tj[3];
    for (int i = 0; i < n; i++) {
      cin>>tj[0]>>tj[1]>>tj[2];
      cout<<find(tj, 1)<<endl;
    }
}